src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -1862,8 +1862,7 @@ void EIRFuelFiredHeatPump::doPhysics(EnergyPlusData &state, Real64 currentLoad)
     auto &thisInletNode = state.dataLoopNodes->Node(this->loadSideNodes.inlet);
     auto &thisOutletNode = state.dataLoopNodes->Node(this->loadSideNodes.outlet);
     auto &sim_component = DataPlant::CompData::getPlantComponent(state, this->loadSidePlantLoc);
-    bool RunFlag = true;
-    if ((this->EIRHPType == DataPlant::PlantEquipmentType::HeatPumpFuelFiredHeating && currentLoad <= 0.0) || !RunFlag) {
+    if (this->EIRHPType == DataPlant::PlantEquipmentType::HeatPumpFuelFiredHeating && currentLoad <= 0.0) {
```

Comment:
The `RunFlag` flag is always `true,` so it is redundant logic and must be removed.